### PR TITLE
update versions and add application http header

### PR DESCRIPTION
**Pull request by mariobodemann**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,7 +3,9 @@ All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## Version [1.1.1][unreleased] - (in development)
-- TBD
+- Changed: Dependency updates
+- Changed: Add application header
```
Comment (dlitvakb) [inline CHANGELOG.md]: I don't see where this is being added, maybe missed adding a file to the commit?

File: src/main/java/com/contentful/generator/Generator.java [hunk 2]
```diff
@@ -110,12 +112,13 @@ public void generate(String spaceId, String pkg, String path, CMAClient client)
...
-   * @param pkg package name for generated classes
-   * @param path package source root
-   * @param token management api access token
+   * @param pkg     package name for generated classes
+   * @param path    package source root
+   * @param token   management api access token
    */
   public void generate(String spaceId, String pkg, String path, String token) {
     CMAClient client = new CMAClient.Builder()
+        .setApplication("Generator.java", "1.1.0")
```
Comment (dlitvakb) [inline src/main/java/com/contentful/generator/Generator.java]: Is there a way to avoid hard coding the version here?

Comment (mariobodemann) [inline src/main/java/com/contentful/generator/Generator.java]: Yes. ;)

File: .travis.yml [hunk 3]
```diff
@@ -1,7 +1,6 @@
 language: java
 
 jdk:
-  - oraclejdk7
```
Comment (loudmouth) [inline .travis.yml]: 👍

File: src/main/java/com/contentful/generator/Generator.java [hunk 4]
```diff
@@ -71,26 +77,26 @@ public void generate(String spaceId, String pkg, String path, CMAClient client)
         String name = contentType.getName();
         if (name != null) {
           String className = normalize(name, CaseFormat.UPPER_CAMEL);
-          models.put(contentType.getResourceId(), className);
+          models.put(contentType.getId(), className);
```
Comment (loudmouth) [inline src/main/java/com/contentful/generator/Generator.java]: ich mag es

File: src/main/java/com/contentful/generator/Generator.java [hunk 5]
```diff
@@ -110,21 +116,33 @@ public void generate(String spaceId, String pkg, String path, CMAClient client)
...
-   * @param pkg package name for generated classes
-   * @param path package source root
-   * @param token management api access token
+   * @param pkg     package name for generated classes
+   * @param path    package source root
+   * @param token   management api access token
    */
   public void generate(String spaceId, String pkg, String path, String token) {
     CMAClient client = new CMAClient.Builder()
+        .setApplication("Generator.java", getVersion())
```
Comment (loudmouth) [inline src/main/java/com/contentful/generator/Generator.java]: gefällt mir besser 😍

**Pull request merged**